overlay/root/Dockerfile.sample

```diff
@@ -0,0 +1,15 @@
+FROM scaleway/ubuntu:amd64-vivid
```

Comment:
I'm not sure I already pushed ubuntu vivid in multiarch, switch to wily which is not deprecated